perl/lib/BarnOwl/Message/Zephyr.pm

```diff
@@ -133,6 +144,14 @@ sub zephyr_cc {
     return undef;
 }
 
+sub zephyr_cc_without_recipient {
+    my $self = shift;
+    my $recipient = lc(strip_realm($self->recipient));
+    my $cc = $self->zephyr_cc;
+    return grep { lc(strip_realm($_)) ne $recipient } split(/\s+/, $cc) if defined $cc;
```

Comment:
You should note in here that the sender is handled in log_filenames -- otherwise it looks like the sender's log file isn't used.